manager/notifier.go

```diff
@@ -453,6 +457,49 @@ func (n *notifier) sendWebhookNotification(op notificationOp, config *pb.Webhook
 	return nil
 }
 
+// Amazon SNS
+func (n *notifier) sendAmazonSnsNotification(op notificationOp, config *pb.AmazonSnsConfig, a *Alert) error {
+	// Create New SNS API
+	snsAPI := sns.New(nil)
+
+	status := ""
+	switch op {
+	case notificationOpTrigger:
+		status = "firing"
+	case notificationOpResolve:
+		status = "resolved"
+	}
+
+	// Create Messages
+	params := &sns.PublishInput{
+		Message: aws.String(fmt.Sprintf("%s -- %s", a.Description, status)),
+		MessageStructure: aws.String("string"),
+		Subject:          aws.String(fmt.Sprintf("%s -- %s", a.Summary, status)),
+		TopicARN:         aws.String(config.GetTopicArn()),
+	}
+
+	resp, err := snsAPI.Publish(params)
+	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok {
+			// Generic AWS error with Code, Message, and original error (if any)
+			fmt.Println(awsErr.Code(), awsErr.Message(), awsErr.OrigErr())
+			if reqErr, ok := err.(awserr.RequestFailure); ok {
+				// A service error occurred
+				fmt.Println(reqErr.Code(), reqErr.Message(), reqErr.StatusCode(), reqErr.RequestID())
+			}
+		} else {
+			// This case should never be hit, the SDK should always return an
+			// error which satisfies the awserr.Error interface.
+			fmt.Println(err.Error())
+		}
+	}
+
+	// Pretty-print the response data.
+	fmt.Println(awsutil.StringValue(resp))
```

Comment:
In general, we don't use `fmt.Println` to output debug information, but the `log.XXX` functions.

In this case, the other notification mechanisms don't output responses either, so I'd just omit this line?